include_core/omrutil.h

```diff
@@ -300,6 +300,21 @@ omr_error_t detectVMDirectory(wchar_t *vmDirectory, size_t vmDirectoryLength, wc
 
 uintptr_t getStorageKey(void);
 
+/* ---------------- zosversion.c ---------------- */
+
+#define ZOS_V2R4_RELEASE 27.00
+#define ZOS_V2R4_VERSION 4
```

Comment:
Suggest adding these (currently defined and used in OpenJ9) and perhaps others from the cited document:
```c
#define ZOS_V1R10_RELEASE 20.00
#define ZOS_V1R10_VERSION 3
```